stf.core/src/stf.core/net/adoptopenjdk/stf/environment/JavaVersion.java

```diff
@@ -83,8 +83,10 @@ private JavaVersion(boolean isPrimaryJvm, String javaHome) throws StfException {
 			throw new StfException("Failed to read result of command: " + javaCmd);
 		}
 
- 		// Eat superfluous first lines (e.g. JIT: env var TR_OPTIONS etc, or JVMJ9VM082E Unable to switch to IFA processor)
-        while (javaVersionOutput.startsWith("JVMJ9VM082E") || javaVersionOutput.startsWith("JIT:")) {
+ 		// Eat superfluous first lines (e.g. JIT: env var TR_OPTIONS etc, or JVMJ9VM082E Unable to switch to IFA processor,
+        // or JVMJ9VM090I Slow response to network query)
+        while (javaVersionOutput.startsWith("JVMJ9VM082E") || javaVersionOutput.startsWith("JIT:")
```

Comment:
It should be more robust to skip anything that starts with "JVMJ9VM" rather than these specific messages. Not sure if there are other possibilities besides these two, and additional ones could be added in the future.

Reply:
I see! Agreed. Updated PR.